packages/block-editor/src/experiments.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	registerAccessToken,
+	getExperimentalAPIs as _getExperimentalAPIs,
+	registerExperimentalAPIs as _registerExperimentalAPIs,
+} from '@wordpress/experiments';
+
+/**
+ * Internal dependencies
+ */
+const ACCESS_TOKEN = {
+	i_realize_my_code_will_break_in_a_few_months_once_the_experimental_apis_are_removed: true,
+};
+registerAccessToken( ACCESS_TOKEN, '@wordpress/block-editor' );
+
+export const registerExperimentalAPIs = _registerExperimentalAPIs.bind(
+	null,
+	ACCESS_TOKEN
+);
+
+export const getExperimentalAPIs = _getExperimentalAPIs.bind(
+	null,
+	ACCESS_TOKEN
+);
```

Comment:
I believe that with some refactoring this could be even simplified to:

```js
const { registerExperimentalAPIs, getExperimentalAPIs } = registerAccessToken( ACCESS_TOKEN, '@wordpress/block-editor' );
```

This way also `@wordpress/experiments` would need only one public method: `registerAccessToken`.

Reply:
Hey that's a good idea!

Reply:
@gziolo I went ahead with that – what do you think?

Reply:
I like it 👍🏻